data-prepper-core/src/main/java/org/opensearch/dataprepper/plugin/PluginPackagesSupplier.java

```diff
@@ -30,8 +30,6 @@ class PluginPackagesSupplier implements Supplier<String[]> {
 
     private static final Logger LOG = LoggerFactory.getLogger(PluginPackagesSupplier.class);
     private static final String DEFAULT_PLUGINS_CLASSPATH = "org.opensearch.dataprepper.plugins";
-    // TODO: Remove this once all plugins have migrated to org.opensearch
```

Comment:
Let's remove this support in Data Prepper 3.0. Please revert these data-prepper-core changes for now.